### PR TITLE
Add SharingUtils with two example for sharing article

### PR DESCRIPTION
**Pull request by Hadidi-007**
Update the README.md to detail the sharing article examples

Jira: https://lekiosk.atlassian.net/browse/PUB-289

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,20 @@ the activity.
 reader.startReaderActivity(this);
 ```
 
-**3. Sample project**
+
+**3 Sharing an article
+
+If your miLibris content has articles, you can implement your own sharing solution by implementing void shareArticleContent. Two examples are provided in the sample project:
```
Comment (MaximeLM) [inline README.md:94]: Suggestion: replace "void shareArticleContent" with "shareArticleContent"

File: app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            .setText(articleUrl)
+        intentBuilder.startChooser()
+    }
+
+    fun shareArticleViaMail(context: Context, article: PdfReader.Article) {
+
+
+        val intent = Intent(Intent.ACTION_SENDTO)
+        intent.data = Uri.parse("mailto:")
+        intent.putExtra(Intent.EXTRA_EMAIL, arrayOf("test@test.com"))
```
Comment (MaximeLM) [inline app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt:35]: You should leave the recipient blank, it's the user that will fill it.

File: app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            context.startActivity(intent)
+        }
+    }
+
+    private fun mailSubject(article: PdfReader.Article): String {
+        // article_title (edition_title, edition_subtitle)
+        return article.title?.let { title ->
+            article.editionTitle.let { editionTitle ->
+                article.editionSubtitle?.let { editionSubtitle ->
+                    "$title  $editionTitle, $editionSubtitle"
```
Comment (MaximeLM) [inline app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt:72]: You should put editionTitle and editionSubtitle between () as in the comment on line 68

File: app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                } ?: "$title  $editionTitle"
```
Comment (MaximeLM) [inline app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt:73]: same here, the edition title should be between parenthesis

**Review by MaximeLM — CHANGES_REQUESTED** [review]

**Review by MaximeLM — APPROVED** [review]

File: app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        activity: Activity,
+        issueMid: String,
+        kioskURL: String,
+        article: PdfReader.Article
+    ) {
+
+        val articleUrl = java.lang.StringBuilder(kioskURL)
+            .append("/share/article")
+            .append("/$issueMid")
+            .append("/${article.mid}").toString()
```
Comment (Ajmir-milibris) [inline app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt:23]:
```suggestion
            "$kioskURL/share/article/$issueMid/${article.mid]"
```

File: app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+        if (intent.resolveActivity(context.packageManager) != null) {
+            context.startActivity(intent)
+        }
+    }
+
+    private fun mailSubject(article: PdfReader.Article): String {
+        // article_title (edition_title, edition_subtitle)
+        return article.title?.let { title ->
+            article.editionTitle.let { editionTitle ->
```
Comment (Ajmir-milibris) [inline app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt:69]: editionTitle doesn't seem to be optional ? 
If that's the case then it's not necessary to use the 'let' on it.

**Review by Ajmir-milibris — APPROVED** [review]

File: app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt (hunk 5 above)
Comment (Hadidi-007) [reply app/src/main/java/com/milibris/pdfreader/sampleapp/sharing/SharingUtils.kt:23]: you need to replace the ] with } so i can commit your suggestion

**Review by Ajmir-milibris — APPROVED** [review]

**Pull request merged**